.github/workflows/update-parset-tests.yml

```
@@ -0,0 +1,65 @@
+name: Update Test262 parser tests
+env:
+  YARN_ENABLE_SCRIPTS: false # disable post-install scripts
+  YARN_NODE_LINKER: pnp # use pnp linker for better performance
+on:
+  workflow_dispatch:
+    inputs: {}
+  schedule:
+    - cron: "0 0 * * 5"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: tc39/test262
+          path: build/test262
+      - name: Use Node.js latest
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: "*"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            yarn-
+      - name: Get latest test262 version
+        id: test262
+        run: echo "::set-output name=sha1::$(sh ./scripts/parser-tests/get-test262-version.sh)"
+      - name: Update test262 commit
+        run: |
+          echo ${{ steps.test262.outputs.sha1 }} | ./scripts/parser-tests/bump-test262-version.sh
+      - name: Build babel parser
+        run: |
+          yarn install --immutable --skip-builds
+          yarn gulp build-rollup
+      - name: Update test262 allow list
+        run: |
+          make test-test262-update-allowlist
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

Since this is not an official action (i.e. `actions/*`) and since we are passing it @babel-bot's access token, I think it would be better to pin it to a specific commit:
```suggestion
        uses: peter-evans/create-pull-request@8c603dbb04b917a9fc2dd991dc54fef54b640b43
```

---

I am working on replacing `create-pull-requests` by actions/github-script.